tests/transport_test.go

```diff
@@ -77,9 +77,9 @@ var _ = Describe("Transport Tests", func() {
 		}
 
 		if insecureRegistry {
-			err = utils.SetInsecureRegistry(f.K8sClient, f.CdiInstallNs, ep())
+			err = utils.SetInsecureRegistry(f.CrClient, ep())
```

Comment:
it is unlikely, but we might want to ensure the CDIConfig is updated correctly before proceeding so we know the insecure registries contain the values we want.

Reply:
check added